### PR TITLE
Release 1.3.0-rc2

### PR DESCRIPTION
**Pull request by adamwalach**
**Description**

Changes proposed in this pull request:

- bumped images to 1.3.0-rc2
- cherry picks for bugfixes

**Related issue(s)**
* https://github.com/kyma-project/kyma/issues/4447

### CODE REVIEW

File: resources/compass/values.yaml [hunk 1]
```diff
@@ -4,20 +4,20 @@ global:
       path: eu.gcr.io/kyma-project/incubator
     connector:
       dir:
-      version: "1.3.0-rc1"
+      version: "1.3.0-rc2"
```
Comment (pkosiec) [inline resources/compass/values.yaml]: Please create PR to compass to its release branch, because these images won't exist

Comment (adamwalach) [reply resources/compass/values.yaml]: https://github.com/kyma-incubator/compass/pull/133

**Pull request merged**